cllc-public-app/ClientApp/src/app/components/applications/tied-house-decleration/tied-house-decleration-form/tied-house-declaration-form.component.html

```diff
@@ -23,6 +23,7 @@
           placeholder="yyyy-mm-dd"
           readonly
           class="form-control"
+          [max]="maxDate"
```

Comment:
optional: you could also define the `[min]` date (if its supported), and specify something like `today - 125 years`.

Technically people have lived that long, even though its incredibly unlikely, even more so to also be on someones liqour licence haha.